modules/ROOT/pages/import-and-export-preferences-studio.adoc

```diff
@@ -0,0 +1,37 @@
+= Export and Import Studio Preferences
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can export your Anypoint Studio preferences and import them into a new version.
+
+Follow this process if you had to download a fresh copy of Studio and you want to recover your previous configurations.
+
+== Export Your Studio Configurations
+
+. In Studio, go to *Preferences*.
+. Ensure that you applied all the configurations that you want to export.
```

Comment:
I'm not sure that the action here is really "Ensure that you applied" so much as "Ensure that the current config has all the features that you want to export."
I mean, would you apply new configs to the existing Preferences if you wanted something MORE, or remove ones that you don't?

Reply:
Oh...okay, now that I see Step 4, I don't think that you need Step 2.

Reply:
This is important because non-applied configurations will not be exported.